src/path/mod.rs

```diff
@@ -750,6 +750,22 @@ pub trait IntoDatabasePath<'a> {
 }
 
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct DatabaseViewPathRef<'a> {
+    db_name: DatabaseNameRef<'a>,
+    view_name: ViewNameRef<'a>,
+}
+
+#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct DatabaseViewPath {
+    db_name: DatabaseName,
+    view_name: ViewName,
+}
```

Comment:
There are other paths below db, like `/db/_temp_view`, that could also be represented by this. But the values are defined in the couch docs, so they could be checked.